cmd/flags/frameworks.go

```diff
@@ -0,0 +1,48 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Framework string
+
+// These are all the current frameworks supported. If you want to add one, you
+// can simply copy and past a line here. Do not forget to also add it into the
+// AllowedProjectTypes slice too!
+const (
+	Chi             Framework = "chi"
+	Gin             Framework = "gin"
+	Fiber           Framework = "fiber"
+	GorillaMux      Framework = "gorilla/mux"
+	HttpRouter      Framework = "httprouter"
+	StandardLibrary Framework = "standard-library"
+	Echo            Framework = "echo"
+)
+
+var AllowedProjectTypes = []string{string(Chi), string(Gin), string(Fiber), string(GorillaMux), string(HttpRouter), string(StandardLibrary), string(Echo)}
```

Comment:
Nit:
To update this list if someone wants to add a new framework they would need to update this in both spots..
Is there a way we can get around this?
Follow - up: is it only these two places or more?

Reply:
> Is there a way we can get around this?

I could try and do something with `iota` and build the slice dynamically I think.


> Follow - up: is it only these two places or more?

You would also need to add the actual implementation logic but besides that these are the only places.

Reply:
I did a little digging and all different implementations I found either also had 2 spots to maintain or weren't type safe